frigate/events.py

```diff
@@ -21,6 +22,12 @@
 logger = logging.getLogger(__name__)
 
 
+class DetectionTypeEnum(str, Enum):
```

Comment:
Does it make more sense to call this `EventTypeEnum`?

Reply:
Probably yeah